app/models/node.rb

```diff
@@ -43,9 +43,7 @@ def self.search(query, order = :default)
           .order(orderParam)
       end
     else
-      nodes = Node.limit(limit)
-        .where('title LIKE ?', '%' + input + '%', status: 1)
```

Comment:
Ah, but we do need this extra title matching -- otherwise we won't match the title against search input. This is just backup code if MySQL/MariaDB are not installed, but I still want to be sure it works. 

I think the issue here is we never defined the `limit` parameter! Not sure how that happened, but if you re-add the original code but remove the `.limit(limit)` we should be OK here. Thanks so much for looking into this!!!